### PR TITLE
Node5 5.11.0 (new formula)

### PR DESCRIPTION
**Pull request by chrmoritz**
This PR adds a formula for node5 to versions, since node is upgraded to v6 in core (Refs: https://github.com/Homebrew/homebrew-core/pull/633).

It is planned to support the node 5 release line over the next 2-3 months (to give people time to migrate to node 6) before it goes eol (because node 5 is not a lts release).

### CODE REVIEW

File: node5.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  url "https://nodejs.org/dist/v5.11.0/node-v5.11.0.tar.xz"
+  sha256 "083766aff72fc5c78f6999dd0d00bc0340a052ab270fd60d6928b719ca2b3fd1"
+  # uncomment once a v5.x staging branch is cut
+  # head "https://github.com/nodejs/node.git", :branch => "v5.x-staging"
+
+  bottle do
+    sha256 "2df9da24389b7b01851b1d794d0ea8ddcce0af2d084233f7382b678defb1a53c" => :el_capitan
+    sha256 "da56a1850d323d090545de46950d986ed29c9cc63efc24b43669f9c6e3effbe7" => :yosemite
+    sha256 "8254c09259aa2d869279f580e489d82181aec17b7555c2f434c290e2625906d1" => :mavericks
+  end
```
Comment (mikemcquaid) [inline node5.rb]: Probably need to scrap the `bottle` for a new formula.

**Comment by mikemcquaid** [comment]
One comment but otherwise LGTM. @DomT4, thoughts?

**Comment by DomT4** [comment]
👍

**Comment by chrmoritz** [comment]
I've removed the `bottle` block.

**Comment by mikemcquaid** [comment]
Thanks again @chrmoritz!

**Pull request closed without merging**